packages/adp-tooling/src/writer/index.ts

```diff
@@ -59,14 +49,15 @@ export async function generate(basePath: string, config: AdpWriterConfig, fs?: E
     if (!fs) {
         fs = create(createStorage());
     }
-    const fullConfig = setDefaults(config);
 
-    fs.copyTpl(join(tmplPath, '**/*.*'), join(basePath), fullConfig, undefined, {
-        globOptions: { dot: true },
-        processDestinationPath: (filePath: string) => filePath.replace(/gitignore.tmpl/g, '.gitignore')
-    });
+    const fullConfig = setDefaults(config);
 
-    await writeUi5Yaml(basePath, fullConfig, fs);
+    if (fullConfig.customConfig?.adp.environment === "C" && fullConfig.flp) {
```

Comment:
Is the FLP config not valid for on prem systems?

Reply:
Yes